src/Command/SqlGeneratorCommand.php

```diff
@@ -0,0 +1,229 @@
+    {
+        $this->setName(self::$defaultName);
+        $this->setDescription('Converts Beans into SQL.');
+        $this->addArgument('destination', \Symfony\Component\Console\Input\InputArgument::REQUIRED, 'Path to folder');
+        $this->addArgument('output', \Symfony\Component\Console\Input\InputArgument::OPTIONAL, 'Output file path');
+    }
+
+    protected function execute(
+        \Symfony\Component\Console\Input\InputInterface $input,
+        \Symfony\Component\Console\Output\OutputInterface $output
+    ) : int
+    {
+        $converted = '';
+        $destination = $input->getArgument('destination');
+        $beans = $this->getBeans($destination);
+
+        foreach ($beans as $bean) {
+            $converted .= $this->generateSqlForBean($bean);
+        }
+
+        $outputFile = $input->getArgument('output');
+
+        if (\is_string($outputFile)) {
+            \file_put_contents($outputFile, $converted);
+        } else {
+            $output->write($converted);
+        }
+
+        return 0;
+    }
+
+    public function generateSqlForBean($className) : string
+    {
+        $bean = new \ReflectionClass($className);
+
+        $toReturn = 'CREATE TABLE `' . \Infinityloop\Utils\CaseConverter::toSnakeCase($bean->getShortName()) . '`(' . \PHP_EOL;
+        $foreignKeys = '';
+        $data = [];
+
+        foreach ($bean->getProperties(\ReflectionProperty::IS_PUBLIC) as $property) {
+            if (!$property->getType() instanceof \ReflectionType) {
+                continue;
+            }
+
+            $data[] = $this->getData($property);
+            $foreignKeys .= $this->getForeignKey($property);
+        }
+
+        $toReturn .= $this->buildTable($data);
+
+        $toReturn .= ($foreignKeys === '' ? '' : \PHP_EOL) . $foreignKeys;
+
+        $toReturn .= ')' . \PHP_EOL;
+        $toReturn .= self::INDENTATION . 'CHARSET = `utf8mb4`' . \PHP_EOL;
+        $toReturn .= self::INDENTATION . 'COLLATE `utf8mb4_general_ci`;' . \PHP_EOL . \PHP_EOL;
+
+        return $toReturn;
+    }
+
+    private function buildTable(array $data) : string
+    {
+        $longestNameLength = $this->getLongestByType($data, 'name');
+        $longestDataTypeLength = $this->getLongestByType($data, 'dataType');
+        $toReturn = '';
+
+        foreach ($data as $row) {
+            $nameLength = \mb_strlen($row['name']);
+            $dataTypeLength = \mb_strlen($row['dataType']);
+            $toReturn .= self::INDENTATION . $row['name'] . \str_repeat(' ', $longestNameLength - $nameLength + 1);
+            $toReturn .= $this->buildDataType($row, $longestDataTypeLength, $dataTypeLength);
+            $toReturn .= $row['notNull'] === true ? ' NOT NULL' : '';
+            $toReturn .= $row['default'];
+            $toReturn .= $this->buildAutoIncrement($row['name']);
+            $toReturn .= ',' . \PHP_EOL;
+        }
+
+        return $toReturn;
+    }
+
+    private function buildAutoIncrement(string $name) : string
```

Comment:
I thought autoincrement is supposed to be in the defaultValue column - so this should be in the getDefault function.